airbyte-integrations/connectors/source-postgres/integration_tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use a base image with PostgreSQL client
+FROM python:3.9
+
+RUN apt-get update && apt-get install -y \
+    libpq-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/* \
+
+WORKDIR /usr/src/app
+
+# Copy the script to the container
+COPY seed/hook.py .
+COPY seed/requirement.txt .
+
+RUN ls .
+
+RUN pip install --no-cache-dir -r ./requirement.txt
```

Comment:
I recommend moving this up above the `COPY seed/hook.py` line so to take advantage of caching. Otherwise it has to rerun when `seed/hook.py` changes.